tools/payloads/configbytes/apk_configbytes_extract.rb

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env ruby
+# -*- coding: binary -*-
+
+class ExtractError < StandardError; end
+class HelpError < StandardError; end
+class UsageError < ExtractError; end
+
+require 'rex'
+require 'tmpdir'
+require 'optparse'
+require 'open3'
+require 'nokogiri'
+require 'fileutils'
+
+def run_cmd(bin, args, verbose)
+  begin
+    path = Rex::FileUtils.find_full_path(bin)
+    cmd = [path, args].join(" ")
+    stdin, stdout, stderr = Open3.popen3(cmd)
```

Comment:
It looks like [`Open3.capture3`](https://www.rubydoc.info/stdlib/open3/Open3.capture3) would be a better solution here. Also the arguments shouldn't be concatenated with a space when they can be directly passed as an array.

Reply:
Tried to use Open.capture3 and I kept getting errors. However, I updated arguments to be passed as an array.